inst/extdata/rgnchn/lookup.txt

```diff
@@ -47,17 +47,16 @@ Jiangsu,42
 Jiangxi,46
 Jilin,39
 Liaoning,38
-Nei Mongol Zizhiqu,37
-Ningxia  Zizhiqu,62
+Inner Mongolia,37
+Ningxia,62
 Qinghai,61
 Shaanxi,59
 Shandong,47
-Shanghai Shi,41
+Shanghai,41
 Shanxi,36
 Sichuan,55
-Tianjin Shi,34
-Xinjiang  Zizhiqu,63
-Xizang  Zizhiqu,58
+Tianjin,34
+Tibet,63
```

Comment:
All the rest of the changes in this file just replace some of the names with renderings that will be more familiar to English-speaking users, but this one swaps the numbering on Tibet and Xinjiang.  Was the numbering on these ARs wrong before?